tinc.yaml

```diff
@@ -0,0 +1,3 @@
+dependencies:
+  - name: riak-protobuf
+    path: protobuf
```

Comment:
@llelf Thanks for pointing us to tinc, definitely worth swapping ideas with them :+1: